### ISSUE TITLE
Critical questions

### ISSUE THREAD

**Issue by ryanbnl** [DEUS-BV/covid19-dp3t-android#4]
* Team of 50 developers

You claim to have a team of 50 developers working on this project; however so far there's only one person committing?

Further your offices are right next to Amsterdam Central and don't look large enough to fit 50 developers. Where are these 50 developers based?

* Long experience building apps for large companies

You claim to have built systems for companies such as Shell. However your company is extremely young; how have you managed to do that?

**Comment by JaapBraam** [comment]
You really don't need a large team to make a great app!
It is very shortsighted to measure the quality of a team by its size. In my experiance, simple projects developed by large teams often don't deliver...

**Comment by ryanbnl** [comment]
Absolutely, I'm not. However the representative of this team was boasting that they had a team of 50+ developers.

Two days ago they were claiming that they developed this solution themselves and were arguing strongly against sharing of source code publically.

Now at the end of the process they're saying how they're customizing an existing project and really want to work with the community to implement it.

They've been extremely misleading and opaque which is why I'm asking direct questions on their Github tracker and not getting any answers.

All of the time we have other teams who we've been ripping to shreds on twitter, who have then taken the effort to get into contact with us. They've then listened to that advise and changed their implementation. Literally the guys you're seeing on the stream engaging with experts in the community directly.
